source/modules/MEI.cmn.xml

```diff
@@ -581,110 +597,54 @@
       <memberOf key="att.startEndId"/>
     </classes>
     <attList>
+      <attDef ident="d" usage="opt">
```

Comment:
@rettinghaus The order of the string attributes is now d, c, b, e, f ... correct?